### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by lblemail**
Add src.cpp and header.h

### CODE REVIEW

File: src.cpp [hunk 1]
```diff
@@ -0,0 +1 @@
+# This is the test source file
```
Comment (divinel) [inline src.cpp]: Add a main function at least!

**Review by divinel — CHANGES_REQUESTED** [review]
Add a main function in src.cpp

**Comment by lblemail** [comment]
main func added

**Review by divinel — APPROVED** [review]

**Comment by divinel** [comment]
Ready to merge

**Pull request merged**